catroid/src/main/java/org/catrobat/catroid/content/Sprite.java

```diff
@@ -545,4 +549,28 @@ public boolean isBackgroundSprite() {
 	public Multimap<EventId, EventThread> getIdToEventThreadMap() {
 		return idToEventThreadMap;
 	}
+
+	public int getUnusedPointer() {
```

Comment:
no usage for this function, could you provide some Info what's the purpose of it?

Reply:
depending which of these two functions you mean
getUnusedPointer provides a unused pointer in the range of 0-19 as there can only be 20 touch events simultaneously
getIdToEventThreadMap was there bevore so your guess might be better than mine

Reply:
`getUnusedPointer()` has no usage as far as I can see? did I miss something?

Reply:
TapAtAction line 47, kotlin removes the "get" from functions

Reply:
pointer = sprite?.unusedPointer ?: 0

Reply:
ach ja, I overlooked it